.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Epic
+about: A larger task consisting of more deliverables
+labels: 'Epic, priority: undecided'
+pipelines: Epics
+---
+
+## Background
+A clear and concise intro into the situation.
+
+## Goal
+The goal that epic wants to achieve.
+
+## Expected Task List [Optional]
```

Comment:
I think that this is not needed since Github/Zenhub will automatically create the list from the issues connected to the epic.

Reply:
Sure